### PR TITLE
Google Map Tweaks

### PR DESCRIPTION
**Pull request by GordonSmith**
Improve marker handling.
Recenter USA

Signed-off-by: Gordon Smith <gordon.smith@lexisnexis.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
@mlzummo Please Review

File: src/map/GMap.js [hunk 1]
```diff
@@ -9,17 +9,33 @@
...
+    };
+
     GMap.prototype.enter = function (domNode, element, d) {
         Graph.prototype.enter.apply(this, arguments);
 
         this._googleMap = new google.maps.Map(d3.select(this._target).node(), {
-            zoom: 3,
-            center: new google.maps.LatLng(41.850033, -87.6500523),
+            zoom: 4,
+            center: new google.maps.LatLng(42.877742, -97.380979),
```
Comment (mlzummo) [inline src/map/GMap.js]: should we think about putting this in testData, so that default center is user selectable?

Comment (GordonSmith) [inline src/map/GMap.js]: Possibly, but need somewhere to default it to and this one is practical at least.

**Comment by mlzummo** [comment]
looks good

**Pull request merged**